packages/suite/src/views/settings/coins/index.tsx

```diff
@@ -18,8 +19,19 @@ const StyledCoinsGroup = styled(CoinsGroup)`
 const Settings = () => {
     const { mainnets, testnets, enabledNetworks, setEnabled } = useEnabledNetworks();
 
+    const { device, isLocked } = useDevice();
+    const isDeviceLocked = !!device && isLocked();
+
     return (
         <StyledSettingsLayout>
+            {isDeviceLocked && (
+                <DeviceBanner
+                    title={<Translation id="TR_SETTINGS_DEVICE_BANNER_TITLE_UNAVAILABLE" />}
+                    description={
+                        <Translation id="TR_SETTINGS_DEVICE_BANNER_DESCRIPTION_UNAVAILABLE" />
```

Comment:
Those can't be the same translation id with those on Device tab because the text doesn't make sense for coins tab.